.github/workflows/dev.yaml

```diff
@@ -0,0 +1,62 @@
+name: Release to dev
```

Comment:
Does it make to move jobs with steps to a single file and execute it with different parameters from dev/stage/prod workflows?

Reply:
> Also, it will be hard to define "on-push" CI/CD for different branches

This part can be implemented either via parameters or be declared in each file as it is now.